hw/ip/adc_ctrl/data/adc_ctrl_sec_cm_testplan.hjson

```diff
@@ -0,0 +1,27 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// Security countermeasures testplan extracted from the IP Hjson using reggen.
+//
+// This testplan is auto-generated only the first time it is created. This is
+// because this testplan needs to be hand-editable. It is possible that these
+// testpoints can go out of date if the spec is updated with new
+// countermeasures. When `reggen` is invoked when this testplan already exists,
+// It checks if the list of testpoints is up-to-date and enforces the user to
+// make further manual updates.
+{
+  testpoints: [
+    {
+      name: sec_cm_bus_integrity
```

Comment:
this countermeasure is covered by the testpoint below. not sure how we will define the relationship between these 2 testpints?  
https://github.com/lowRISC/opentitan/blob/199d45626f8a7ae2aef5d9ff73793bf9a4233711/hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson#L58-L65

Reply:
IMO duplication is fine (as we discussed offline) - user just needs to wire up the test `"{name}_tl_intg_err"` to this testpoint as well, which is a very small extra work.